packages/quantic/cypress/e2e/default-2/result-list/folded-result-list/folded-result-list-expectations.ts

```diff
@@ -30,8 +31,8 @@ function logFoldingActionEvent(
   const {title, uri, position, documentId} = result || {};
   cy.wait(eventName)
     .then((interception) => {
-      const analyticsBody = interception.request.body;
-      const customData = analyticsBody?.customData;
+      const analyticsBody = getAnalyticsBodyFromInterception(interception);
+      const customData = analyticsBody.customData;
```

Comment:
`req.body` can be null no? 

in common-expectations.ts :
`analyticsBody = req.body;` if `req.body` is missing, it's possible that `analyticsBody.customData` will throw? Is that what we want?

Reply:
Either it throws here, or fails the test couple lines below. Pretty much same end result.